### PR TITLE
ensure React + ReactDOM are no longer in the toolkit bundle

### PR DESCRIPTION
**Pull request by peter-mouland**
**Before**
```
toolkit.js  724 kB
```
**After**
```
toolkit.js  49.9 kB 
```

I've also updated the demo code to only import the react component rather than the whole toolkit.  This is to our recommended method rather than importing the whole toolkit.

### CODE REVIEW

File: docs/webpack.config.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ module.exports = {
     path: './docs/dist/',
     filename: 'docs.js'
   },
+  externals: {
```
Comment (tomatau) [inline docs/webpack.config.js]: should mention that now these are declared as externals, react and reactDom need to be available as global names for the dist bundle to work.

**Comment by bentidy** [comment]
What's the benefit of importing the uncompiled components in the docs?

If we keep it using the compiled js it's an extra check to verify that it works.

@ldabiralai

**Comment by ldabiralai** [comment]
This should now fix the issue we were having earlier which caused us to open  #80, so I'll go ahead and close that for now, we can always reopen it if we still have the issue. 

Thanks for helping us with this @peter-mouland and good spot on React being included twice :+1:

**Comment by bdo** [comment]
:+1:

**Comment by nowarkdev** [comment]
@alexjfno1 @TristanCooper :+1:

**Pull request merged**